README.md

```diff
@@ -41,6 +41,8 @@ Starting DeskKit SDK version `4.0`, prebuilt frameworks are attached in github r
 3. With your project open in Xcode, select your Target. Under General tab, find Embedded Binaries and then click the + button.
 4. Click the Add Other... button, navigate to the framework files (`DeskKit.framework`, `DeskAPIClient.framework`, `DeskCommon.framework`) and select them. Check the Destination: Copy items if needed checkbox when prompted.
 
+**IMPORTANT: Attached prebuilt frameworks contains a binaries which has been built for a number of architectures `(x86_64, i386, armv7, arm64)`. According to [this radar](http://www.openradar.me/radar?id=6409498411401216) before submission to AppStore you must strip off simulator slices `(x86_64, i386)`.**
```

Comment:
If it's multiple frameworks this should be: "Attached prebuilt frameworks contain binaries which have been built..."
If it's one framework this should be: "Attached prebuilt framework contains binaries which have been built..."